### PR TITLE
Create 2021-1-25-Metabolomics-Mini-Symposium.md

### PR DESCRIPTION
**Pull request by foellmelanie**

### CODE REVIEW

File: _events/2021-1-25-Metabolomics-Mini-Symposium.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  email: foellmelanie@gmail.com
+location: online
+external: https://docs.google.com/document/d/1F3gKicQ7MaMn-EKwC3cJbhf7s8-j4RYtdjEJQH0EeK0/preview
+---
+
+Join us for the __Galaxy Metabolomics Mini-Symposium__ on __Friday 29th January from 9 am to 12 pm CET__ on zoom.
+
+Learn about recent developments and future plans in the Galaxy Metabolomics community. Get to know other 
+Galaxy metabolomics researchers and join discussions about the perspectives, interests and needs of Galaxy 
+users and tool developers. In addition to a thematic exchange we hope that this event will foster new
```
Comment (beatrizserrano) [inline _events/2021-1-25-Metabolomics-Mini-Symposium.md:18]:
```suggestion
users and tool developers. In addition to a thematic exchange, we hope that this event will foster new 
```

File: _events/2021-1-25-Metabolomics-Mini-Symposium.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+site: freiburg
+tags:
+title: "Galaxy Metabolomics Mini-Symposium"
+starts: 2021-01-29
+ends: 2021-01-29
+organiser:
+  name: Melanie Föll
+  email: foellmelanie@gmail.com
+location: online
+external: https://docs.google.com/document/d/1F3gKicQ7MaMn-EKwC3cJbhf7s8-j4RYtdjEJQH0EeK0/preview
```
Comment (beatrizserrano) [inline _events/2021-1-25-Metabolomics-Mini-Symposium.md:11]: external link means that only the first paragraph will be previewed, left @bgruening?

Comment (bgruening) [reply _events/2021-1-25-Metabolomics-Mini-Symposium.md:11]: yes, and only in the preview, that is probably not what we want, do we?

Comment (beatrizserrano) [reply _events/2021-1-25-Metabolomics-Mini-Symposium.md:11]: yeah so might be better to embed the link in the body

**Pull request merged**